ui/apps/platform/src/hooks/useURLPagination.ts

```diff
@@ -1,11 +1,11 @@
 import { useCallback } from 'react';
-import useURLParameter from './useURLParameter';
+import useURLParameter, { Action } from './useURLParameter';
```

Comment:
Reading `historyAction?: Action | undefined` several times below, what do think is benefit to cost of `HistoryAction` as the type?

To prevent ambiguity with similar-but-different type `type PageActions = 'create' | 'edit' | 'clone';`

Reply:
Works for me!